Header head
uint32 packet_counter
uint32 sample_fine_time